vaccine_feed_ingest/runners/md/arcgis/normalize.py

```diff
@@ -0,0 +1,435 @@
+        "Sun 10am - 5pm, Mon - Fri 8am - 9pm, Sat 9am - 6pm"
+        "Sun 10am - 5pm, Mon - Fri 9am - 9pm, Sat 9am - 6pm"
+        "Sun 10am - 6pm, Mon - Fri 8am - 9pm, Sat 9am - 6pm"
+        "Sun 9am - 5pm, Mon - Fri 8am - 10pm, Sat 9am - 7pm"
+        "Sun 9am - 5pm, Mon - Fri 8am - 9pm, Sat 9am - 7pm"
+        "Wed - Sat 10am to 6pm"
+        """
+        return raw
+
+
+def _pieces(raw: str) -> List[str]:
+
+    """
+    "Mon- Fri 7 a.m. - 11 a.m. and 2 PM-4 PM "
+    """
+    if not raw:
+        return []
+
+    match = re.search(
+        fr"(?P<days>{DAY_OF_WEEK}\s*-\s*{DAY_OF_WEEK})\s*(?P<time1>{TIME_RANGE})\s*and\s*(?P<time2>{TIME_RANGE})",
+        raw,
+    )
+    if match:
+        return [
+            f"{match.group('days')} {match.group('time1')}",
+            f"{match.group('days')} {match.group('time2')}",
+        ]
+
+    """
+    "Mon, Tues, Thurs, Fri: 7:30am - 3pm, Wed: 3pm - 7pm"
+    """
+    comma_sep_days_with_time = f"({DAY_OF_WEEK}((,|and) {DAY_OF_WEEK})*: {TIME_RANGE})"
+
+    """
+    "Sun 10am - 4pm, Mon - Fri 10am - 7pm, Sat 10am - 5pm"
+    "Sun 10am - 5pm, Mon - Fri 8am - 9pm, Sat 9am - 6pm"
+    "Sun 10am - 5pm, Mon - Fri 9am - 9pm, Sat 9am - 6pm"
+    "Sun 10am - 6pm, Mon - Fri 8am - 9pm, Sat 9am - 6pm"
+    "Sun 9am - 5pm, Mon - Fri 8am - 10pm, Sat 9am - 7pm"
+    "Sun 9am - 5pm, Mon - Fri 8am - 9pm, Sat 9am - 7pm"
+    "Mon - Tues 9 a.m. - 1 p.m., Wed - Thurs 2 p.m. - 6 p.m., Fri - Sat 11 a.m. - 3 p.m."
+    "Mon - Fri 8:30 a.m. - 6:30 p.m. Sat. 9 a.m. - 1 p.m."
+    """
+    day_maybe_range_with_time = f"({DAY_OF_WEEK}( - {DAY_OF_WEEK})? {TIME_RANGE})"
+
+    for pattern in (comma_sep_days_with_time, day_maybe_range_with_time):
+        if re.search(pattern, raw):
+            return [m[0] for m in re.findall(pattern, raw)]
+
+    """
+    "24 Hours"
+    "Mon - Fri 7 a.m. - 11 a.m."
+    "Mon - Fri 7 a.m. - 7 p.m."
+    "Mon - Fri 7 am - 7 pm"
+    "Mon - Fri 7am - 7 pm"
+    "Mon - Fri 8 a.m. - 4:30 p.m."
+    "Mon - Fri 8 a.m. - 5 p.m. "
+    "Mon - Fri 8 a.m. - 5 p.m."
+    "Mon- Fri 7 a.m. - 11 a.m."
+    "Wed - Sat 10am to 6pm"
+    "Monday through Saturday, from 10 a.m. - 12 p.m."
+    "Weds -Sun 10 a.m. - 2 p.m."
+    "each day 10 a.m. - 2 p.m."
+    """
+    return [raw]
+
+
+def _normalize_hours(raw: str) -> Tuple[str, str]:
+    match = re.search(
+        r"(?P<hour>\d\d?)(:(?P<minute>\d\d))?\s*(?P<am_pm>(a|p)\.?m\.?)", raw
+    )
+    hour = int(match.group("hour"))
+    if (1 <= hour <= 11) and (match.group("am_pm").startswith("p")):
+        hour += 12
+    minute = int(match.group("minute") or "0")
+    return datetime.time(hour, minute).isoformat("minutes")
+
+
+def _normalize_days(raw: str) -> List[str]:
+    potentials = {
+        "sun": "sunday",
+        "mon": "monday",
+        "tues": "tuesday",
+        "wed": "wednesday",
+        "thur": "thursday",
+        "thurs": "thursday",
+        "fri": "friday",
+        "sat": "saturday",
```

Comment:
Don't forget that you can use [these enums](https://github.com/CAVaccineInventory/vaccine-feed-ingest-schema/blob/0a020a0e101d5c97ecd06a9ace15a4d6d798be2d/vaccine_feed_ingest_schema/location.py#L159-L166) to avoid string typos (e.g., `schema.DayOfWeek.MONDAY`)

Reply:
Ooh, should have looked for that.  Thanks!